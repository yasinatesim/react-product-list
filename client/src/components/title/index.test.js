/* eslint-disable */
import React from 'react';
import renderer from 'react-test-renderer';
import Title from './index';

describe('title component', () => {
  it('normal case', () => {
    const tree = renderer.create(<Title>Text</Title>).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('no children', () => {
    const tree = renderer.create(<Title />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
