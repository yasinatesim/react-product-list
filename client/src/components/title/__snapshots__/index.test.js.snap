// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`title component no children 1`] = `
<div
  className="sc-AxjAm beupUC"
>
  <h3 />
</div>
`;

exports[`title component normal case 1`] = `
<div
  className="sc-AxjAm beupUC"
>
  <h3>
    Text
  </h3>
</div>
`;
